src/engraving/dom/score.cpp

```diff
@@ -178,6 +178,8 @@ Score::Score(MasterScore* parent, bool forcePartStyle /* = true */)
     m_masterScore = parent;
     if (DefaultStyle::defaultStyleForParts()) {
         m_style = *DefaultStyle::defaultStyleForParts();
+
+        checkChordList();
```

Comment:
Actually, maybe it's preferable to move the `checkChordList()` calls out of this if/else statement, and put it just below `m_style.precomputeValues();`.